src/remoteRunner.js

```diff
@@ -94,7 +94,13 @@ async function uploadStaticPackage({
     return assetsDataRes.path;
   } catch (e) {
     if (e.statusCode !== 404) {
-      throw e;
+      logger.warn(
```

Comment:
I think we need to make this change in more places, like domRunner.js

Reply:
You are right.